pkg/cmd/adm/register_member.go

```diff
@@ -36,6 +36,8 @@ const (
 	TokenExpirationDays = 3650
 )
 
+var invalidKubeConfigError = errors.New("invalid kubeconfig file")
```

Comment:
only a detail - is there any reason for defining it as a global var?

Reply:
I think it is a recommended practice. See https://github.com/Djarvur/go-err113?tab=readme-ov-file#reports (which is a linter we don't use ATM).

Reply:
ok, but why? I mean, what is the benefit of it, and what is the disadvantage of doing it the other way `return fmt.Errorf(...)` as we do all over the place?
I can be wrong, but most of the projects do exactly it - initialize the error when returning them.

Reply:
I think if done using a `var`, you can use it with `errors.Is()`, which becomes more handy when the error is public.